src/main/capi/prepared-c.cpp

```diff
@@ -159,6 +159,49 @@ duckdb_logical_type duckdb_param_logical_type(duckdb_prepared_statement prepared
 	return nullptr;
 }
 
+idx_t duckdb_prepared_column_count(duckdb_prepared_statement statement) {
+	auto wrapper = reinterpret_cast<PreparedStatementWrapper *>(statement);
+	if (!wrapper || !wrapper->statement || wrapper->statement->HasError()) {
+		return 0;
+	}
+	return wrapper->statement->ColumnCount();
+}
+
+const char *duckdb_prepared_column_name(duckdb_prepared_statement statement, idx_t field_idx) {
```

Comment:
Nit: let's call this `column_idx` instead of `field_idx`. Same for the other functions.

Reply:
Thank you for pointing that out. I've updated the variable name to column_idx as suggested. The same change has been applied to the other functions.